tool/tctl/common/user_command.go

```diff
@@ -57,10 +57,12 @@ type UserCommand struct {
 
 // Initialize allows UserCommand to plug itself into the CLI parser
 func (u *UserCommand) Initialize(app *kingpin.Application, config *service.Config) {
+	const helpPrefix string = "[Teleport DB users only]"
```

Comment:
What are these users called in docs? I usually refer to them as "local users".

Reply:
@benarent what do you think of `Local users only` vs `Teleport DB users only`?